experimental/devices/ina219/ina219.go

```diff
@@ -157,7 +157,7 @@ func (d *Dev) calibrate(sense physic.ElectricResistance, maxCurrent physic.Elect
 	d.mu.Lock()
 	defer d.mu.Unlock()
 
-	d.currentLSB = maxCurrent / (2 << 15)
+	d.currentLSB = maxCurrent / (1 << 15)
 	d.powerLSB = physic.Power(d.currentLSB * 20)
```

Comment:
this is another error of mine too:
should be 
```d.powerLSB = physic.Power((maxCurrent*20 + (1 << 14)) / (1 << 15))```
as it int rounding cause it to be 1953120 and not 1953125 as it should be with the defaults.